include/psim/fc/orbit_controller.yml

```
@@ -0,0 +1,33 @@
+
+name: OrbitControllerInterface
+type: Model
+comment: >
+    Interface for how the flight computer's orbit controller will interact with the rest of the simulation
+
+args:
+    - satellite
+    - other
+params:
+    - name: "truth.follower.m"
+      type: Real
+adds:
+    - name: "truth.follower.deltaV"
+      type: Real
+gets:
```

```suggestion
args:
    - satellite
    - other

params:
    - name: "truth.{satellite}.m"
      type: Real

adds:
    - name: "truth.{satellite}.delta_v"
      type: Real

gets:
```
Couple of comments.

First we have the two arguments `satellite` and `other` going which is awesome but we should use them to mangle the names of the parameters and fields we're adding here as well to keep the orbit controller model as portable as possible -- i.e. we could easily flip it to make the leader chasing the follower.

Second, field names are normally prefixed with a fixed string depending on what set of models is responsible for adding them to the simulation. In this case that means we should name anything being added by the orbit controller with `fc.**`. Also, just as a general naming convention, we don't have any fields that use camel case or capital letters in psim so I'd recommend renaming the `**.deltaV` field to something like `**.cummulative_dv` or something like that.

Lastly, we should add a comment field to added state fields. If someone if looking for a description of what's going on the interface file is going to be where they're looking.